extensions/sql-migration/src/controllers/mainController.ts

```diff
@@ -0,0 +1,42 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import { Controller } from '../models/controller';
+
+export class MainController implements Controller {
```

Comment:
Do we really need to do this controller stuff? It never really seems worth the extra layer of abstraction

Reply:
That's true, I guess we don't really need it. It kinda makes dealing with disposables easier but yeah I'll try to think of a cleaner way of doing this.